SQL 26th sep/26sep'24.txt

```diff
@@ -0,0 +1,78 @@
+1.) 
+create database employee2;
+use employee2;
+CREATE TABLE Employees (  EmployeeID INT PRIMARY KEY,FirstName VARCHAR(50), LastName VARCHAR(50),  Department VARCHAR(50),Salary DECIMAL(10, 2),HireDate DATE);
+select * from Employees;
+insert into Employees values (1,'aaa','Abin','Marketing',15000, '2002-07-20');
+insert into Employees values (2,'bbb','beni','sales ', 60000, '2020-08-01');
+insert into Employees values (3,'ccc','Cyril','IT',87000, '2019');
```

Comment:
what will be the output of this insert ? Will this work ?

Reply:
it will be getting the values mentioned in query inserted to 3 rows in the employees table. yes, it works.